test/LULESH-v1.0/LULESH-v1.0_RAJA-variants/luleshRAJA-parallel-FT.cxx

```diff
@@ -2756,7 +2756,7 @@ int main(int argc, char *argv[])
    RAJA::Timer timer_main;
    RAJA::Timer timer_cycle;
 
-   timer_main.start();
+   timer_main.start("timer_main");
```

Comment:
@rhornung67 this is the change that is pervasive, these functions taking in a const char*. If you're not using Caliper, this just calls an ordinary start/stop, otherwise it uses Caliper, which takes advantage of the name info